src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -339,13 +347,15 @@ private void debug(String message, Throwable e) {
     private String formatMessage(ILoggingEvent loggingEvent) {
 
         JsonObject logMessage = formatMessageAsJson(loggingEvent.getTimeStamp(), loggingEvent.getLevel().levelStr,
-                loggingEvent.getFormattedMessage(), loggingEvent.getLoggerName(), loggingEvent.getThreadName(), Optional.ofNullable(loggingEvent.getThrowableProxy()));
+                loggingEvent.getFormattedMessage(), loggingEvent.getLoggerName(), loggingEvent.getThreadName(),
```

Comment:
You format using parameters from the logging event but then you pass the logging event it self which is strange. I would pass the ThrowableProxy.